server/config/default.json

```diff
@@ -0,0 +1,43 @@
+{
+  "host": "localhost",
+  "port": 3030,
+  "public": "../public/",
+  "paginate": {
+    "default": 10,
+    "max": 50
+  },
+  "authentication": {
+    "entity": "user",
+    "service": "users",
+    "secret": "iHDlrsCynzOx/jOl8/VssQnbS5s=",
```

Comment:
Is this something we should be committing? Not sure what this is but looks like a secret key :)

Reply:
I'd have to look up specifically whats happening here, I'm guessing it's probably used for hashing the passwords or some sort of extra area of authentication, we can change it later for production from whats in the repo, if you want I can make it more obviously not the final key

Reply:
Ok sounds good. Just wanted to make sure it's something we want to be committing.